README.md

```diff
@@ -2,131 +2,14 @@
-All these issues can lead to potential money losses of companies. Static analysis of programs helps to reduce the number of such bugs and issues 
-that cannot be found by validations on the compiler's side.
-
-There are different kinds and purposes of static analysis: it can be simple analysis using AST (abstract syntax tree), it can be more complex CFA
-(control-flow analysis), interprocedural analysis, context sensitive analysis, e.t.c. Static analyzers can check code style, find potential issues on the runtime in
-the logic of an application, check for code smells and suggest best practices. But what exactly should static analyzers do? How their functionality can be measured?
-What is an acceptance criteria for Which functionality do developers really need when they are writing a brand new analyzer? These questions are still remain not answered,
-even after decades of development of static analyzers. 
-
-## Problematics
-Each and every creator of static analyzers in the beginning of his development journey starts
-from the very simple thing: types of issues that his tool will detect. This leads to a searching of existing lists of potential issues or test packages that can be used to 
-measure the result of his work or can be used for TDD (test driven development). In other areas of system programming such benchmarks and test sets already exists,
-for example [SPEC.org](http://spec.org/benchmarks.html) benchmarks are used all over the world to test the functionality, evaluate and measure the performance of different applications
-and hardware: from compilers to CPUs, from web-servers to Java Clients. But there are no test sets and even strict standards for detection of issues that can be found in
-popular programming languages. There were some guidelines of coding on C/C++ done by [MISRA](https://www.misra.org.uk/), but there are no analogues of it even for the most popular
-languages in the world like Python and [Java](https://stackoverflow.com/questions/6050618/is-there-a-java-equivalent-to-misra-c).
- 
-In this situation each and every new developer that reinvents his new code style or mechanism of static analysis each time reinvents his brand new test framework and writting test sets
-that have been written already thousands of times for his analyzer/linter. Someone uses existing guidelines like [Google code style](https://google.github.io/styleguide/javaguide.html)
-or using [PMD rules](https://pmd.github.io/). But in all cases lot of time will be spent on reinventing, writing and debuging tests.
-
 ## What is SAVE?
-SAVE - is an eco-system for measuring, testing and certification of static code analyzers. Instead of writing your test framework SAVE will provide you a command line application with a
+[SAVE](https://github.com/cqfn/save) - is an eco-system for measuring, testing and certification of static code analyzers. Instead of writing your test framework SAVE will provide you a command line application with a
```

Comment:
I guess the link has changed to `https://github.com/cqfn/save-cloud`

Reply:
Here it's intentionally a link to main repo with SAVE CLI, but I forgot to change links in badges, thanks for noticing.